packages/taquito-sapling/src/sapling-state/interface.ts

```diff
@@ -6,3 +8,34 @@ export interface SaplingStateTree {
 }
 
 export type MerkleTree = undefined | string | [string, MerkleTree, MerkleTree];
+
+export interface SpendDescription {
```

Comment:
I think we can move the type SpendDescription, OutputDescription, Ciphertext, and SaplingTransaction in a types.ts file in the src folder. Those types will be reused in different classes (like the SaplingTransactionBuilder)

Reply:
That's a good suggestion, will move